msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam-errors.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Errors associated with Web Account Manager (WAM)
+description: "When building applications using MSAL.NET and the Web Account Manager (WAM), developers may run into issues. This article explains potential errors and mitigations."
+ms.date: 08/30/2023
+---
+
+# Errors associated with Web Account Manager (WAM)
+
+Generally, errors generated by the MSAL component that interacts with the Web Account Manager (WAM) are [automatically converted](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/d25f45f7e356bd92986e412dd68b5d341343aa6a/src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs#L56) to an instance of <xref:Microsoft.Identity.Client.MsalException>. This enables developers to not worry about the internals and instead handle errors through idiomatic .NET constructs. However, for additional deep dives, developers will often need to investigate specific error messages.
```

Comment:
Add a link to the main exceptions doc? https://review.learn.microsoft.com/en-us/entra/msal/dotnet/advanced/exceptions